ggshield/cmd/secret/scan/precommit.py

```diff
@@ -40,7 +40,7 @@ def check_is_merge_with_conflict(cwd: Path) -> bool:
 
 @click.command()
 @click.option(
-    "--merge-skip-unchanged",
+    "--skip-unchanged-merge-files",
     "-m",
```

Comment:
Not exactly related, but think we should not have a short option for this. Users are unlikely to write this command by hand, so there is no productivity gain there. When using the option in a script file or in a .pre-commit-config.yaml, the long option is more self-documenting.

Reply:
Good point thanks :) I'll remove it.